### DISCUSSION TITLE
Add the possibility to put html code in autoreply email.

### DISCUSSION THREAD

**Discussion by Zathiel** [Mailu/Mailu#3249] [Feature requests / Ideas]
Dear Mailu team,

In https://github.com/Mailu/Mailu/issues/3220 I asked about the possibility to put some HTML code when autoreply is activated.

The reason is my team is currently working with html / images footer and we need that for our leave and so on messages

If you need anything feel free to reach me

Regards

**Comment by nextgens** [comment]
Just to be clear: you can already do this... but you cannot use Mailu's web-interface for doing it; you need to configure a sieve script with the vacation module yourself.

https://stackoverflow.com/questions/56754555/what-is-the-correct-way-to-format-a-sieve-script-to-include-mime-with-html

You can edit the template Mailu uses at core/admin/mailu/internal/templates/default.sieve

The difficulty with providing the feature you ask for is the UX. Most HTML emails have both a text/plain and an HTML part... we would need to ask the user for two versions of his ooo message. We could of course use a WYSIWYG control for the HTML part... but then do we also ask for text/plain, do we suggest the recipient should get a better MUA or do we attempt an auto-conversion? A lot of questions have to be answered/specced out before we could consider implementing it.

IMHO it's a lot of complexity for little gain.
